apps/api/src/services/external/apiNodeService.ts

```diff
@@ -66,7 +107,13 @@ export async function getChainStats() {
     totalSupply: result.totalSupply,
     communityPool: result.communityPool,
     inflation: result.inflation,
-    stakingAPR: (result.inflation * (1 - result.communityTax)) / (result.bondedTokens / result.totalSupply)
+    stakingAPR:
+      result.bondedTokens && result.inflation && result.communityTax && result.totalSupply
+        ? (result.bondedTokens > 0
+          ? (result.inflation * (1 - result.communityTax)) /
+          (result.bondedTokens / result.totalSupply)
+          : null)
+        : undefined,
```

Comment:
this nested ternaries are really hard to understand. Could you please organize this into a flow with a couple of descriptive vars?